docs/compass/03-pairing-adapter.md

```diff
@@ -0,0 +1,19 @@
+# Pairing Adapter 
+
+In case of Application managed by the Integration System, One Time Token can be fetched not from Compass Connector,
+but rather from External Service that allows pairing Integration System and Application.
+For that purpose, the Pairing Adapter component was created.
+Pairing Adapter is highly configurable via environment variables. Thanks to that it is possible to translate request from Director to any external system.  
+If the Integration System defines Pairing Adapter, the Director passes all information regarding given Application to Adapter to generate a token.
+To configure Pairing Adapter for Integration System, edit Configmap `pairing-adapter-config`.
+This configmap has the following structure:
+
+```config.json: '{"{integration system id}":"{pairing adapter URL}"}'```
+
+for example:
+
+```config.json: '{"0c4bfacf-2237-40f1-ab3d-8e35582e26f2":"http://compass-pairing-adapter/adapter"}'```
+
+Communication between Director, Pairing Adapter and External Token Service is presented in the following diagram:
+
+![](./assets/pairing-adapters.svg)
```

Comment:
Please explain the steps under the diagram with the ordered list. What does point no. 6 stand for? Another thing, please add the actor that performs action from step no.1. Correct the capitalization as follows: `ConfigMap`, `one-time token`, `response`. Round the connectors and move connector descriptions and steps above/below the connectors themselves. Last but not least: `Management Plane cluster` vs. `Compass cluster`? So far, we used the second one in the docs. 

When correcting the diagram, please refer to the [guidelines](https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/02-diagrams.md).